articles/mfa/guides/configure-phone.md

```diff
@@ -0,0 +1,128 @@
+
+Optionally, you can [customize your SMS or Voice notification templates](/mfa/guides/customize-phone-messages).
+
+![MFA SMS Settings](/media/articles/mfa/sms-settings.png)
+
+## Message Delivery Method
+
+You can choose if you want to give users the option of getting text messages, voice calls, or both.
+
+## Twilio configuration
+
+If you choose to deliver SMS via Twilio, follow these steps to configure your SMS factor.
+
+1. Open an account with Twilio. You will need a [Twilio Account SID](https://www.twilio.com/help/faq/twilio-basics/what-is-an-application-sid) and a [Twilio Auth Token](https://www.twilio.com/help/faq/twilio-basics/what-is-the-auth-token-and-how-can-i-change-it). These are the Twilio API credentials that Auth0 will use to send an messages to the user. 
+
+  You may also need to enable permissions for your geographic region for [SMS](https://support.twilio.com/hc/en-us/articles/223181108-How-International-SMS-Permissions-work) and [Voice](https://www.twilio.com/console/voice/calls/geo-permissions). If you use Voice, your Twilio phone number needs to be enabled to make Voice calls.
+
+2. Configure the connection. Enter your **Twilio Account SID** and **Twilio Auth Token** in the appropriate fields.
+
+3. Choose your **SMS Source**.
+
+  * If you choose **Use From**, you will need to enter the **From** phone number that users will see as the sender. You may also configure this in Twilio. 
+
+  * If you choose **Use Messaging Services**, you will need to enter a [Messaging Service SID](https://www.twilio.com/docs/sms/services/services-send-messages).
+
+  If you are using Voice, you always need to configure 'From' even if you are using 'Messaging Services' for SMS. Make sure the phone number is configured to send both SMS and Voice messages.
+
+5. Click **Save**.
+
+## Custom Phone Messaging providers
+
+Phone Messaging providers not currently integrated with Auth0 can be supported by using the [Send Phone Message](/hooks/extensibility-points/send-phone-message) Hook. To learn how to implement this in your MFA flow, check the examples for different providers below:
+
+* [Amazon SNS](/mfa/send-phone-message-hook-amazon-sns)
+* [Twilio](/mfa/send-phone-message-hook-twilio)
+* [Infobip](/mfa/send-phone-message-hook-infobip)
+* [TeleSign](/mfa/send-phone-message-hook-telesign)
+* [Vonage](/mfa/send-phone-message-hook-vonage)
+* [Esendex](/mfa/send-phone-message-hook-esendex)
+* [Mitto](/mfa/send-phone-message-hook-mitto)
+
+## Using the Management API to configure Voice or SMS
+
+You can use the Management API to configure which Message Delivery Methods are enabled by using the `/api/v2/guardian/factors/phone/message-types` endpoint. 
+
+The `messages_types` parameter is an array that can have ["sms"], ["voice"], or ["sms", "voice"]. You need a [Management API Token](https://auth0.com/docs/api/management/v2/tokens) with the `update:guardian_factors` scope as a Bearer Token to call the API:
```

Comment:
Should we change formatting to make it consistent with other parts of the documentation?

old: ["sms"], ["voice"], or ["sms", "voice"].
new: `["sms"]`, `["voice"]`, or `["sms", "voice"]`